bin/poly

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env ruby
 
-require_relative('../lib/polyphony/http')
+require File.expand_path('../lib/polyphony/http/server', __dir__)
+require File.expand_path('../lib/polyphony/http/server/rack_adapter', __dir__)
 
 app_path = ARGV.first || './config.ru'
-app = Polyphony::HTTP::Rack.load(app_path)
+app = Polyphony::HTTP::Server::RackAdapter.load(app_path)
```

Comment:
Let's be careful with all those nested namespaces! ;-) I think it should stay `Rack` rather than `RackAdapter`, since it's not an adapter like the HTTP1 or HTTP2 ones.

Reply:
Because we are using `instance_eval` in `load`, using `Rack` can create ambiguity issues, both in the internal code of the library (such as [here](https://github.com/digital-fabric/polyphony-http/blob/master/lib/polyphony/http/server/rack.rb#L42)), and in the [consumer code](https://github.com/digital-fabric/polyphony-http/blob/master/examples/cuba.ru#L6) when referring to `Rack`.

Reply:
That's fine, let's leave it like that